### PR TITLE
Turn back on style checks in CI

### PR DESCRIPTION
**Pull request by white238**
whoops.

### CODE REVIEW

**Review by jamiebramwell — APPROVED** [review]
whoops indeed! Thanks for catching this!

**Comment by white238** [comment]
this is gonna take a bit. clangformat got dropped from the new docker images on accident.  I'm doing a full rebuild from the base image up.

**Comment by white238** [comment]
This is ready for review.

File: scripts/azure-pipelines/linux-check.sh [hunk 1]
```diff
@@ -33,11 +33,18 @@ if [[ "$DO_COVERAGE_CHECK" == "yes" ]] ; then
     cmake_args="$cmake_args -DENABLE_COVERAGE=ON -DGCOV_EXECUTABLE=/home/serac/gcov"
 fi
 
+if [[ "$DO_STYLE_CHECK" == "yes" ]] ; then
+    cmake_args="$cmake_args -DENABLE_CLANGFORMAT=ON -DCLANGFORMAT_EXECUTABLE=/usr/bin/clang-format"
+    echo "~~~~~ START: clang executables ~~~~~"
+    ls -al /usr/bin | grep clang
+    echo "~~~~~ END: clang executables ~~~~~"
```
Comment (joshessman-llnl) [inline scripts/azure-pipelines/linux-check.sh]: Was this debugging snippet intentionally left in? Separately I wonder if there's a way of checking somewhere else that the style checks actually ran

**Review by joshessman-llnl — APPROVED** [review]

File: scripts/azure-pipelines/linux-check.sh (hunk 1 above)
Comment (white238) [reply scripts/azure-pipelines/linux-check.sh]: In my late night coding it seemed like a good idea at the time.  removed for a more targeted erroring out.

**Pull request merged**